src/transforms/aws_cloudwatch_logs_subscription_parser.rs

```diff
@@ -1,7 +1,7 @@
 use std::iter;
 
 use serde::{Deserialize, Serialize};
-use shared::aws_cloudwatch_logs_subscription::{
+use vector_common::aws_cloudwatch_logs_subscription::{
     AwsCloudWatchLogsSubscriptionMessage, AwsCloudWatchLogsSubscriptionMessageType,
 };
```

Comment:
I _think_ it's because both VRL's stdlib uses it for its functions, but it's also supported by some (perhaps deprecated?) transform, and/or is used in some of the ongoing Datadog pipelines work.

But I could be wrong there.